port/integration/resource_test.go

```diff
@@ -121,13 +125,52 @@ func TestPortIntegrationBasic(t *testing.T) {
 	})
 }
 
+func TestPortIntegrationPatchTitleNull(t *testing.T) {
+	integrationIdentifier := utils.GenID()
+	installationAppType := "kafka"
+	err := os.Setenv("PORT_BETA_FEATURES_ENABLED", "true")
+	if err != nil {
+		t.Fatal(err)
+	}
+	var testPortIntegrationResourceBasic = createIntegration(integrationIdentifier, installationAppType)
+
+	var testAccBaseIntegrationPermissionsConfigUpdate = strings.Replace(testPortIntegrationResourceBasic, "\"my-kafka-cluster\"", "null", -1)
```

Comment:
Why you call it `PermissionsConfigUpdate`, copy paste mistake?

Reply:
nice catch, oopis